core/templates/dev/head/pages/exploration-editor-page/statistics-tab/templates/playthrough-modal.template.html

```diff
@@ -8,9 +8,19 @@ <h3 class="modal-title">Sample Playthrough</h3>
   </div>
   <div class="row oppia-pt2" style="margin-top: 0.5rem;">
     <div ng-repeat="displayBlock in reversedDisplayBlocks" style="text-align: left; margin-left: 0.2rem;">
-      <div ng-if="!isDisplayBlockOnInitDisplay(displayBlock)" ng-bind-html="renderBlockHtml(displayBlock, createDisplayBlockNavId(displayBlock))" id="remainingActions<[playthroughIndex]><[getDisplayBlockIndex(displayBlock)]>" style="display: none;"></div>
+      <div ng-if="!isDisplayBlockOnInitDisplay(displayBlock)" id="remainingActions<[playthroughIndex]><[getDisplayBlockIndex(displayBlock)]>" style="display: none;">
```

Comment:
We typically avoid using id attrs. Where are you making use of this / do you actually need it?

Reply:
@seanlip This has been existing since the original implementation. We use the ID attribute to hide the display blocks that should not be displayed. Clicking the button to view more learner actions triggers a function that displays the display block with the corresponding ID.

Reply:
Hm, I see. Yeah, there was a few things in the original implementation that should probably not have gone in (e.g. constructing html in JS) :P

Is there a way to do what you want to do without using ids?

Reply:
@seanlip I guess we can achieve the same with classes and then using getElementsByClass instead of getElementsById. I'll try that.

Reply:
Done.